configure.ac

```diff
@@ -0,0 +1,76 @@
+dnl require autoconf 2.60 (AS_ECHO/AS_ECHO_N)
+AC_PREREQ([2.60])
+define(_CLIENT_VERSION_IS_RELEASE, false)
+define(_COPYRIGHT_YEAR, 2015)
+AC_INIT([Armory],m4_esyscmd([grep "^BTCARMORY_VERSION" armoryengine/ArmoryUtils.py | cut -d "(" -f2 | cut -d ")" -f1 | tr -d '[[:space:]]' | tr ',' '.']))
+AC_CONFIG_SRCDIR([cppForSwig/BlockObj.cpp])
+AC_CONFIG_AUX_DIR([build-aux])
+AC_CONFIG_MACRO_DIR([build-aux/m4])
+
+AC_CANONICAL_HOST
+
+dnl faketime breaks configure and is only needed for make. Disable it here.
+unset FAKETIME
+
+case $host in
+    *mingw*)
+        TARGET_OS=windows
+        ;;
+    *darwin*)
+        TARGET_OS=darwin
+        ;;
+    *linux*)
+        TARGET_OS=linux
+        ;;
+    *)
+        ;;
+esac
+
+dnl Automake init set-up and checks
+AM_INIT_AUTOMAKE([no-define subdir-objects foreign])
+
+# Enable debug
+AC_ARG_ENABLE([debug],
+    [AS_HELP_STRING([--enable-debug],
+                    [use debug compiler flags and macros (default is no)])],
+    [enable_debug=$enableval],
+    [enable_debug=no])
+
+dnl Define variables
+AC_ARG_VAR([PYVER], [Defines the version of Python to use (e.g. python2.7)])
```

Comment:
I tried making it like the PYVER in the original cppForSwig/Makefile. It
seemed like it was designed to specify say python2.6 instead of python. But
as you say 2.6 support is gone, but it may be useful still for python 3
(python3.3 instead of python3). But I'll look at that link and see about it.